### PR TITLE
Non-US locale warning for proftpd

### PR DESCRIPTION
**Pull request by szepeviktor**
For example with Hungarian default locale proftpd logs:
```
ápr 06 16:56:30 szerver.szepe.net proftpd[31424] szerver.szepe.net (94-*.pool.digikabel.hu[94.*]): FTP session closed.
```

### CODE REVIEW

**Comment by szepeviktor** [comment]
That is a very old finding but remained unshared.

File: config/filter.d/proftpd.conf [hunk 1]
```diff
@@ -2,6 +2,9 @@
 #
 # Set "UseReverseDNS off" in proftpd.conf to avoid the need for DNS.
 # See: http://www.proftpd.org/docs/howto/DNS.html
+# When the default locale for your system is not en_US.UTF-8
+# be sure to add this to /etc/default/proftpd
```
Comment (yarikoptic) [inline config/filter.d/proftpd.conf]: This is a Debian specific location, thus better to phrase so it is obvious

```

# These regular expressions devised for logs when locate is en_US.UTF-8.
# If your default locale is different, run proftpd under en_US.UTF-8 locale, e.g.
# on Debian-based systems add this to /etc/default/proftpd:
# export LC_TIME="en_US.UTF-8"
```
?

Comment (szepeviktor) [inline config/filter.d/proftpd.conf]: Added.

**Comment by szepeviktor** [comment]
@yarikoptic When do you expect the next release?

**Comment by yarikoptic** [comment]
A month ago :-) but never got enough time to polish out remaining little issues and cook the release. Interested to help?

**Comment by szepeviktor** [comment]
No. I am sorry.

**Pull request merged**